### PR TITLE
Add mobile layout for checkout page

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What problem is this solving?
Adds support for mobile resolutions in the new checkout.

[Related clubhouse story](https://app.clubhouse.io/vtex/story/32812/layout-da-p%C3%A1gina-de-checkout-no-io)

#### How should this be manually tested?
[Workspace](https://chkio--checkoutio.myvtex.com/checkout/#)

#### Checklist/Reminders

- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Clubhouse story (if applicable).

#### Screenshots or example usage
new-checkout-responsive

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

**Review by ArthurSampaio — APPROVED** [review]

File: store/blocks/container-mobile.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "flex-layout.row#container-mobile": {
+    "children": ["flex-layout.col#checkout-mobile"],
+    "props": {
+      "blockClass": "checkoutContainerMobile",
+      "paddingTop": 6
+    }
+  },
+
```
Comment (darlenedms) [inline store/blocks/container-mobile.json:9]: Is this empty line intentional?

Comment (lucasecdb) [reply store/blocks/container-mobile.json:9]: yes, it was to visually separate the blocks, but it doesn't help much. I just find it too difficult to understand the structure of the page by looking at JSON files and tried to separate them better, but its still really awful at visualizing it :-(

Comment (lucasecdb) [reply store/blocks/container-mobile.json:9]: but I can remove it if you want!

Comment (darlenedms) [reply store/blocks/container-mobile.json:9]: No, that's fine. I'm not well versed on "blocks", I have the same difficulties to understand these giant JSONs but want to make sure if there is any type of standard related to empty lines.

**Review by darlenedms — APPROVED** [review]

**Review by jeffersontuc — APPROVED** [review]
LGTM

**Pull request merged**